irods/test/modules/test_client_exit_functions.py

```diff
@@ -0,0 +1,44 @@
+import irods.at_client_exit
+from irods.at_client_exit import (register as register_prc_exit_function,
+                                  BEFORE_PRC_CLEANUP,
+                                  DURING_PRC_CLEANUP,
+                                  AFTER_PRC_CLEANUP)
+#  Used in test of:
+#    irods.test.data_obj_test.TestDataObjOps.test_proper_execution_of_client_exit_functions__issue_614
```

Comment:
Does this comment apply to the entire python file? If so, perhaps it should be placed above the import statements.

Reply:
Done.